polarCoordinateTraversal.m

```diff
@@ -149,37 +157,96 @@
 ray_circle_vector = [circle_center(1) - ray_origin(1); circle_center(2) - ray_origin(2)]';
 v = dot(ray_circle_vector,ray_unit_vector);
 
+% II. Calculate Voxel ID Theta.
```

Comment:
We should check to see if the ray is at the origin first and then
```else if``` so that we aren't calling ```atan2``` twice here.